### DISCUSSION TITLE
How to use Debugoutput property to log Email SMTP Console of Send Action?

### DISCUSSION THREAD

**Discussion by AvIsBeastMC** [PHPMailer/PHPMailer#2243] [Q&A]
Hey!
Umm I recently got to know that you can log the SMTP Console of the Send action using Debugoutput property, though I don't know how to use and make it work.

**Comment by Synchro** [comment]
There's [an example in the code](https://github.com/PHPMailer/PHPMailer/blob/master/src/PHPMailer.php#L404) of using either a callable you write yourself, or a PSR-3-compatible logger such as [monolog](https://packagist.org/packages/monolog/monolog), which you would use like this:

```php
$log = new Logger('name');
$log->pushHandler(new StreamHandler('path/to/your.log', Logger::WARNING));
$mail->Debugoutput = $log;
```
